test/integration/test_generate_specs.py

```
@@ -79,12 +77,14 @@ def test_generate_specs_max_min() -> None:
                     if fn.name == "get_min_max":
                         get_min_max_src.add(fn.get_source_code_with_specs())
 
-
-            assert len(get_min_max_src) > 0, f"Expected at least one verified specification for 'get_min_max'"
+            assert len(get_min_max_src) > 0, (
+                f"Expected at least one verified specification for 'get_min_max'"
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)